views/components/ChartList.jsx

```diff
@@ -34,13 +35,26 @@ const ChartList = () => {
       console.error(error, '*****')
     }
   }
+  const onDuplicateChart = async (id) => {
+    try {
+      const duplicated = await duplicateChart(id)
+      if (duplicated?.data) {
+        await getCharts().then(({ data }) => {
+          setChartList(data)
+        })
+      }
+    } catch (error) {
+      console.error(error, '*****')
```

Comment:
should we remove this prior to merging? We can still find the error in the network tab for debugging

Reply:
What do you think about a future implementation of error handling through the UI? If error display toast message.

Reply:
That sounds like a good feature to add 👍

Reply:
![Uploading Screen Shot 2022-08-08 at 2.13.53 PM.png…]()

Reply:
👍 I think we could still remove this log statement in the meantime, up to you!

Reply:
https://github.com/AMP-SCZ/dpdash/pull/127/commits/169ebf209173726ca5b878bc32344e6c71aa2403